pkg/knuu/instance.go

```diff
@@ -522,6 +523,11 @@ func (i *Instance) Start() error {
 	i.state = Started
 	logrus.Debugf("Set state of instance '%s' to '%s'", i.k8sName, i.state.String())
 
+	err = i.WaitInstanceIsRunning()
+	if err != nil {
+		return fmt.Errorf("error waiting for instance '%s' to be running: %w", i.k8sName, err)
```

Comment:
is `%w` okay without simple quotes?

Reply:
`%w` wraps the error, you usually don't have quotes there.

The error message later looks like: `error bla this: error bla that: error bla huh`